TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/Shooter.java

```diff
@@ -0,0 +1,22 @@
+package org.firstinspires.ftc.teamcode.subsystems;
+
+import com.arcrobotics.ftclib.command.SubsystemBase;
+import com.arcrobotics.ftclib.hardware.motors.Motor;
+import com.arcrobotics.ftclib.hardware.motors.MotorEx;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+public class Shooter extends SubsystemBase {
+    private Motor kuunav;
+
+    public Shooter(HardwareMap hmap, String name){
+        kuunav = new Motor(hmap, name);
+    }
```

Comment:
Pass a motor object, not hardware map and string name. There is no point, just pass in the motor.